front/src/assets/operationStudies/ciChOperationalPointsOnDPYMAS.ts

```diff
@@ -0,0 +1,174 @@
+type CiChOperationalPoints = {
+  [key: string]: string[];
+};
+
+const CI_CH_OPERATIONAL_POINTS_ON_DPY_MAS: CiChOperationalPoints = {
```

Comment:
This data structure is the result of preprocessing the list of OPs for the DPY-MAS line provided in  [Perigny-MiramasFRET_all_prs.xlsx](https://github.com/user-attachments/files/17636347/Perigny-MiramasFRET_all_prs.xlsx). I chose this representation because it is more concise and simplifies filtering.